### PR TITLE
remove hard-coded-solar-filename

### PR DESCRIPTION
**Pull request by YakelynRJ**
This PR removes the hard-coded FORCING_DIR (imported from config). Instead of using FORCING_DIR to check the if statements, a full path file_name is used. getdata.py was cleanup a bit but is essentially the same. 

Coverage reports (updated automatically):
- test_unit: [83%](https:\/\/output.circle-artifacts.com\/output\/job\/67987b43-781e-48c6-90c1-f7aa7e81d25a\/artifacts\/0\/tmp\/coverage\/htmlcov-test_unit\/index.html)

### CODE REVIEW

File: external/radiation/python/radiation_astronomy.py [hunk 1]
```diff
@@ -50,7 +49,7 @@ def __init__(self, me, isolar, solar_filename):
                     " with cycle approximation (old values)!",
                 )
 
-            file_exist = os.path.isfile(os.path.join(FORCING_DIR, self.solar_fname))
```
Comment (brianhenn) [inline external/radiation/python/radiation_astronomy.py:53]: So is this needed because the `solar_fname` being provided already has the forcing dir now?

**Review by brianhenn — COMMENTED** [review]
Happy to approve but maybe you can explain why this was needed a little more clearly?

File: external/radiation/python/radiation_astronomy.py (hunk 1 above)
Comment (YakelynRJ) [reply external/radiation/python/radiation_astronomy.py:53]: FORCING_DIR is imported from config so there were conflicts when I used a different path.

**Review by nbren12 — COMMENTED** [review]
Sorry I forgot to hit "submit"....

File: external/radiation/python/getdata.py [hunk 2]
```diff
@@ -334,40 +334,43 @@ def astronomy(forcing_dir, isolar, tile_number):
...
-                data = xr.open_dataset(os.path.join(forcing_dir, solar_file))
+            file_name = "solarconstant_noaa_an.nc"
+            solar_file = os.path.join(forcing_dir, file_name)
+            if os.path.isfile(solar_file):
+                data = xr.open_dataset(solar_file)
             else:
                 warnings.warn(f'Requested solar data file "{solar_file}" not found!',)
                 raise FileNotFoundError(
                     " !!! ERROR! Can not find solar constant file!!!"
                 )
```
[the remark this answers is not in the document]
Comment (YakelynRJ) [reply external/radiation/python/getdata.py:356]: I just modified to read at the end of the if statements

**Review by brianhenn — APPROVED** [review]
@YakelynRJ LGTM

**Pull request merged**